docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+As of v0.10.0, Knative started to release Knative operators as tools to install, configure and manage Knative. This
+guide will explain how to install and uninstall Knative with Knative operators.
+
+There are two major components in Knative: Serving and Eventing. There are two operators: [Knative Serving Operator](https://github.com/knative/serving-operator)
+and [Knative Eventing Operator](https://github.com/knative/eventing-operator), respectively dedicated to each of them. Please make sure you have set up a Kubernetes
+cluster accessible to your local workstation.
+
+## Limitations of Knative Operators:
+
+Knative operators are still in Alpha phase. They only focus on Knative installation for the generic Kubernetes platform,
+which means Docker-Desktop Kubernetes on Mac and Minikube can be directly used as the Kubernetes clusters for you to install
+Knative with Knative operators, without any additional configuration out of the scope of the operator custom resource,
+but vendor-specific Kubernetes services, like Google Kubernetes Engine, IBM Kubernetes Service, OpenShift, etc, may need
+some additional configurations to work after the Knative is installed by the operator. In short, the Knative operators of
+the open source version only supports the following Kubernetes services, without additional configurations:
+
+   * Docker-Desktop Kubernetes on Mac
+   * Minikube
```

Comment:
What about `kubeadm`-created clusters?

More generally, why aren't other platforms supported?

Reply:
I would like say if we install operator directly on vendor-specific platforms, we need to some other customization to make it work. The customization/additional configuration vary from platform to platform.